### PR TITLE
Add a recipe for R package ENMeval

### PR DESCRIPTION
**Pull request by ColineRoyaux**
Checklist
- [ ] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [ ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [ ] Source is from official source.
- [ ] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [ ] Build number is 0.
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [ ] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

File: recipes/r-enmeval/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    - r-raster
+
+test:
+  commands:
+    - $R -e "library('ENMeval')"           # [not win]
+    - "\"%R%\" -e \"library('ENMeval')\""  # [win]
+
+about:
+  home: https://CRAN.R-project.org/package=ENMeval
+  license: GPL-3
```
Comment (dbast) [inline recipes/r-enmeval/meta.yaml:49]:
```suggestion
  license: GPL-2.0-or-later
```

**Comment by ColineRoyaux** [comment]
Thanks @dbast !

File: recipes/r-enmeval/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  home: https://CRAN.R-project.org/package=ENMeval
+  license: GPL-2.0-or-later
+  summary: Automatically partitions data into evaluation bins, executes ecological niche models
+    across a range of settings, and calculates a variety of evaluation statistics.  Current
+    version only implements ENMs with Maxent (Phillips et al. 2006) or maxnet (Phillips
+    et al. 2017).
+  license_family: GPL3
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/GPL-3'
+    - LICENSE
```
Comment (dbast) [inline recipes/r-enmeval/meta.yaml:57]: There seems to be no license in the tarball, which is fine.
```suggestion
    - LICENSE
```

**Comment by ColineRoyaux** [comment]
Recipe generated with conda_r_skeleton_helper

**Comment by ColineRoyaux** [comment]
@dbast Ok, thanks so much for all the help ! I'm a first-timer ;)

**Comment by dbast** [comment]
Thanks!

**Pull request merged**